llvm/include/llvm/TargetParser/PPCTargetParser.def

```diff
@@ -141,23 +149,30 @@ PPC_LNX_CPU("power10",47)
   #define AIX_BUILTIN_PPC_TRUE 1
   #define AIX_BUILTIN_PPC_FALSE 0
   #define USE_SYS_CONF 2
+  #define SYS_CALL 3
 
   // Supported COMPARE_OP values.
   #define COMP_EQ  0
+  #define COMP_GT 1
+  #define COMP_GE 2
+  #define COMP_NE 3
```

Comment:
Can we not omit this and use `CmpInst::Predicate` in the function-style macros (thereby eliminating the need for the switch statement that uses them as well)?

Reply:
good idea.